AllReadyApp/Web-App/AllReady/Areas/Admin/Features/Tasks/EditTaskCommandHandler.cs

```diff
@@ -56,16 +56,25 @@ public EditTaskCommandHandler(AllReadyContext context)
             }
 
             // Add new attachment
-            if (message.Task.NewAttachment != null && !string.IsNullOrEmpty(message.Task.NewAttachment.Name))
+            if (message.Task.NewAttachment != null && !string.IsNullOrEmpty(message.Task.NewAttachment.FileName))
```

Comment:
I can remove some of it, we do need some check here though, since adding a new attachment is optional